### PR TITLE
Add some more Permission-Strings

### PR DESCRIPTION
**Pull request by Springjunky**
I have add some more permission-Strings and a warning, that not always every permission is available.

Here is our checklist for contributors. No hard requirement here, just a reminder

- [x] Please describe what you did

- [x] Link to issue you're working on if there's a relevant one

- [x] Did you provide a test-case to demonstrate feature actually works / issue is fixed ?

- [x] Please also consider adding a line to [CHANGELOG.md](https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/CHANGELOG.md)

### CODE REVIEW

File: demos/global-matrix-auth/README.md [hunk 1]
```diff
@@ -19,6 +19,27 @@ jenkins:
...
+        - "Run/Delete:authenticated"
+        - "Run/Replay:authenticated"
+        - "Run/Update:authenticated"
+        - "SCM/Tag:authenticated"
         - "Overall/Read:anonymous"
         - "Overall/Administer:authenticated"
 ```
+
+Some permissions depends on actual plugin-usage. 
+For Example: "Release/*:authenticated" is only available if you _use_ the Releaseplugin in one of your jobs.
```
Comment (casz) [inline demos/global-matrix-auth/README.md]: add another newline above as suggested by codacy :smile:
The example is not that readable when viewing it.

https://github.com/jenkinsci/configuration-as-code-plugin/blob/e6ff1680bcd11a61617a70606be89da13c95832a/demos/global-matrix-auth/README.md

Comment (casz) [reply demos/global-matrix-auth/README.md]:
```suggestion
For Example: "Release/*:authenticated" is only available if you _use_ the Release plugin in one of your jobs. 
```

**Comment by Springjunky** [comment]
Sorry i am totally confused now .. is there a step missing to get this pull request merged ?

**Comment by casz** [comment]
I went to bed, while waiting for Jenkins CI reporting back with a successful check :smile:

**Pull request merged**